devices/grlib-multi/test-grlib-multi.c

```diff
@@ -775,6 +775,5 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
-	UnityMain(argc, (const char **)argv, runner);
-	return 0;
+	return UnityMain(argc, (const char **)argv, runner) == 0 ? EXIT_SUCCESS : EXIT_FAILURE;
```

Comment:
The problem is still not resolved, please check the following code and just run in on host:

```C
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
	return 256 == 0 ? EXIT_SUCCESS : EXIT_FAILURE;
}
```

Reply:
In the scenario you provided, it will always return 1, because of EXIT_FAILURE

Reply:
oh, that's right - it seems to work fine